components/elm/bld/namelist_files/use_cases/20thC_CMIP6_transient.xml

```diff
@@ -38,6 +38,7 @@
 <finidat hgrid="ne30np4" mask="oEC60to30v3">lnd/clm2/initdata_map/20180316.DECKv1b_A1.ne30_oEC.edison.clm2.r.1980-01-01-00000.8575c3f_c20190904.nc</finidat>
 
 <flanduse_timeseries hgrid="r05">lnd/clm2/surfdata_map/landuse.timeseries_0.5x0.5_hist_simyr1850-2015_c240308.nc</flanduse_timeseries>
+<flanduse_timeseries hgrid="r025" sim_year_range="1850-2000"  use_crop=".false." >lnd/clm2/surfdata_map/landuse.timeseries_0.25x0.25_hist_simyr1850-2015_c240125.nc</flanduse_timeseries>
```

Comment:
The file name suggests its 1850-2015 but "sim_year_range" is 1850-2000.   Is one of those wrong?

Reply:
@rljacob, Great catch. I will fix the range.

Reply:
I edited the commit and forced pushed the branch.